.gitignore

```diff
@@ -254,3 +254,9 @@ aws-cdk-web/
 # Create React App
 build/
 types/
+
+# Esbuild
+
+dist/esbuild.wasm
+public/esbuild.wasm
+lib/construct/cdk-web-lambda/go-wasm.js
```

Comment:
this is ignored here, but also included in this diff? it should not be here I think.
I think the ignore path is incorrect